ramls/voucher.raml

```diff
@@ -51,3 +53,28 @@ resourceTypes:
         collection-item:
           schema: voucher
           exampleItem: !include acq-models/mod-invoice-storage/examples/voucher.sample
+  /voucher-lines:
+    type:
+      collection:
+        schemaCollection: voucherLineCollection
+        schemaItem: voucherLine
+        exampleCollection: !include acq-models/mod-invoice-storage/examples/voucher_line_collection.sample
+        exampleItem: !include acq-models/mod-invoice-storage/examples/voucher_line.sample
+
+    get:
+      description: Get list of voucher lines
+      is: [
+        pageable,
+        searchable: {description: "with valid searchable fields: for example poLineId", example: "externalAccountNumber==\"567891045\""}
```

Comment:
poLineId? It does not exist in voucher schema.

Reply:
Changed it